.gitignore

```diff
@@ -7,3 +7,6 @@ node_modules
 
 # third party deps
 /third_party/
+
+# Rollup TypeScript cache
+/.rpt2_cache/
```

Comment:
Is there a way this could be put into `out/Debug` using `root_out_dir` in GN ?

Reply:
Before I moved it to tmp, I put this here, I need to remove it now.